assets/js/components/setup/SetupUsingProxyWithSignIn/Splash.js

```diff
@@ -24,31 +24,28 @@ import punycode from 'punycode';
 /**
  * WordPress dependencies
  */
-import { createInterpolateElement } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import { getQueryArg } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
 import { useSelect } from 'googlesitekit-data';
-import WelcomeSVG from '@/svg/graphics/welcome.svg';
-import WelcomeAnalyticsSVG from '@/svg/graphics/welcome-analytics.svg';
-import Link from '@/js/components/Link';
-import ActivateAnalyticsNotice from '@/js/components/setup/ActivateAnalyticsNotice';
-import CompatibilityChecks from '@/js/components/setup/CompatibilityChecks';
+import { useFeature } from '@/js/hooks/useFeature';
+import LegacySplashContent from '@/js/components/setup/SetupUsingProxyWithSignIn/LegacySplashContent';
+import RefreshedSplashContent from '@/js/components/setup/SetupUsingProxyWithSignIn/SplashContent';
```

Comment:
Let's just call this `SplashContent`, we don't need to prefix it with `Refreshed`.


```suggestion
import SplashContent from '@/js/components/setup/SetupUsingProxyWithSignIn/SplashContent';
```